Gruntfile.js

```diff
@@ -186,6 +186,8 @@ module.exports = function(grunt) {
           'sentinel/src/lib/pupil/**',
           'build/**',
           'config/**',
+          'api/src/migrations/convert-translation-messages.js',               // until jshint
+          'api/tests/integration/migrations/convert-translation-messages.js'  // supports await/async
```

Comment:
I don't like excluding the whole files from the jshint execution as it may hide other bugs that aren't to do with async/await.

I think it's probably safest to just stick to Promises until jshint sorts it out (or we migrate off jshint).